adapters/telegram/src/bot.ts

```diff
@@ -144,7 +144,7 @@ export class TelegramBot<C extends Context = Context, T extends TelegramBot.Conf
 
   async $getFile(filePath: string) {
     if (this.local) {
-      return await this.ctx.http.file(filePath)
+      return await this.ctx.http.file(`file://${filePath}`)
```

Comment:
Should `file:///` be used?

Reply:
Actuall it should be `pathToFileURL`.

Reply:
Perhaps we can determine the prefix by determining whether the first character is `/` or not.

Reply:
> Actuall it should be `pathToFileURL`. I'll fix it.

Is #78 still relevant? I mean if it is, maybe you should not directly use Node APIs.

Reply:
> Is #78 still relevant? I mean if it is, maybe you should not directly use Node APIs.

`pathToFileURL` can be ponyfilled in the browser.

https://github.com/cordiverse/ponyfills/blob/main/packages/url/src/index.ts